tests/optimize/test_qng.py

```diff
@@ -104,6 +104,7 @@ def circuit(params):
         assert np.isclose(cost1, expected_cost)
         assert np.isclose(cost2, expected_cost)
 
+    @pytest.mark.skip("The QNG optimizer is not implemented for split arguments yet.")
```

Comment:
is this a breaking change in behaviour?

Reply:
Yes, because `metric_tensor` now un-stacks the classical Jacobian in this case:
Advantage: `metric_tensor` behaves the same way, independently of the interface and shape of the QNode arguments
Disadvantage: Breaking change and need to update QNG.

Reply:
Good to know - definitely requires a section in the "breaking changes" part of the changelog, as well as a before and after code example!